packages/vector_graphics_compiler/lib/src/geometry/path.dart

```diff
@@ -315,9 +315,10 @@ class PathBuilder implements PathProxy {
 
   /// Adds a rectangle to the new path.
   void addRect(Rect rect) {
-    lineTo(rect.top, rect.right);
+    moveTo(rect.top, rect.right);
```

Comment:
This was drawing funky rects before, will add a test

Reply:
I think this may have worked when we guaranteed the top left corner was always at 0,0. But since we're flattening out transforms that is no longer the case and we need to explicitly list out all moves

Reply:
Yeah we'll need to keep track of the current point :\